### ISSUE TITLE
IPv6 Support

### ISSUE THREAD

**Issue by alantamer** [PackeTsar/radiuid#33]
Hi,

Is there any roadmap for IPv6 Support (both for actual operation on the network and for pushing IPv6 mappings in UID pushes)?

I've seen this issue (https://github.com/PackeTsar/radiuid/issues/1) that says about IPv6 support on version 2, but on my xml conf (version 2.4.2) there is only "Framed-IP-Address" and nothing about "Framed-IPv6-Address".

**Comment by PackeTsar** [comment]
Yes I had planned to build in full IPv6 support but it had fallen by the wayside. I will put this back on the table for development.

**Comment by alantamer** [comment]
Nice, it would be great.

More and more people are using IPv6 now and this feature will help a lot in the IPv6 deployment.

**Comment by alantamer** [comment]
Hello, any improvement on this issue or a at least a prediction?

**Comment by jorsov07** [comment]
+1  for this feature! Just started using IPv6 and hoping this product will support it as well
